### PR TITLE
New package FastTransforms

### PR DESCRIPTION
**Pull request by eschnett**

### CODE REVIEW

File: var/spack/repos/builtin/packages/fasttransforms/package.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    complexity of O(n log n), while the multivariate algorithms are 2-normwise
+    backward stable with a runtime complexity of O(nd+1), where n is the
+    polynomial degree and d is the spatial dimension of the problem."""
+
+    homepage = "https://github.com/MikaelSlevinsky/FastTransforms"
+    url      = "https://github.com/MikaelSlevinsky/FastTransforms/archive/v0.3.4.tar.gz"
+
+    version('0.3.4', sha256='a5c8b5aedbdb40218521d061a7df65ef32ce153d4e19d232957db7e3e63c7e9b')
+
+    variant('quadmath', default=False)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/fasttransforms/package.py]: Needs a description

File: var/spack/repos/builtin/packages/fasttransforms/package.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        install(join_path('src', 'dprk_source.h'), prefix.include)
+        install(join_path('src', 'drop_precision.h'), prefix.include)
+        install(join_path('src', 'fasttransforms.h'), prefix.include)
+        install(join_path('src', 'ftinternal.h'), prefix.include)
+        install(join_path('src', 'ftutilities.h'), prefix.include)
+        install(join_path('src', 'ftutilities_source.h'), prefix.include)
+        install(join_path('src', 'hierarchical_source.h'), prefix.include)
+        install(join_path('src', 'tdc.h'), prefix.include)
+        install(join_path('src', 'tdc_source.h'), prefix.include)
+        install(join_path('src', 'tridiagonal_source.h'), prefix.include)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/fasttransforms/package.py]:
```suggestion
    install('src/*.h', prefix.include)
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/fasttransforms/package.py (hunk 1 above)
Comment (eschnett) [reply var/spack/repos/builtin/packages/fasttransforms/package.py]: Done.

File: var/spack/repos/builtin/packages/fasttransforms/package.py (hunk 2 above)
Comment (eschnett) [reply var/spack/repos/builtin/packages/fasttransforms/package.py]: Done.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**